source/features/update-pr-from-base-branch.tsx

```diff
@@ -53,6 +53,7 @@ function createButton(): JSX.Element {
 }
 
 async function addButton(mergeBar: Element): Promise<void> {
+	// TODO: Drop after July 2025
 	if (elementExists(canNativelyUpdate)) {
```

Comment:
You can add `[aria-label="Update branch options"]` inside `canNativelyUpdate` instead